css/root.css

```diff
@@ -2,7 +2,7 @@
 
 :root {
```

Comment:
You have only included font-weight 400. So all your font-weight variations look the same.

Reply:
Thank you, @venkateshwarans for letting me know. I have includes all variations that google font has. This is how it looks, 
![image](https://user-images.githubusercontent.com/56081584/153043697-ae26935b-1f8e-4971-aa28-8bb63c0aa9e2.png)